src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -62,6 +60,8 @@ public static string GetFullPath(string path)
 
             return result;
         }
+        
+        public static char[] GetInvalidFileNameChars() => new char[] { '\0', '/' };
```

Comment:
The members already aren't in alphabetical order (scroll down in the file to find GetTempPath, IsPathRooted, below methods like RemoveRelativeSegments, etc.)